Online-VS-Offline-IDE.md

```diff
@@ -0,0 +1,38 @@
+# Online vs Offline IDEs
+
+## Offline IDEs
+
+There are several offline [IDE](Integrated Development Environment) that programmers use in order to stay more productive to learn and be more productive at work.
+
+- **[Eclipse](https://www.app5.unisys.com/offerings/ClearPathConnection/eclipse.html)** is more than a Java editor. Its biggest advantages are code completion by tabbing method which saves a lot of time when writing documentation.
+
+  It has a build in syntax check to fix any mistype words while coding a project. All of features we've come to expect of an IDE are their code completion, templates, integration  with various SCMSs, and integration with build systems.
+
+  Its code formatting and cleanup tools are very powerful. I find its build system to work well and intuitively. I think these are the things upon which its reputation is really built.
+
+  Furthermore it has a refactoring that find and replace function, variables and classes of a specific work.
+
+  Eclipse offers more plugins, platforms, frameworks, and is simpler to use compare  to other IDEs such as **IntelliJ**, **BlueJ** and **JEdit**.
+
+- **[NetBeans](https://netbeans.org/features/)** is a free intuitive editor that does it all compared from other IDEs. It has simple Swing GUI design tool to build user interfaces by dragging and dropping components for instance buttons, textboxes.
+
+  The _disadvantage_ it is that with all its features it is slow to load and uses  more memory than other IDE's.
+
+- **[IntelliJ IDEA](https://www.jetbrains.com/idea/)** it is the most efficient offline editor however it is not free. It is faster than most editors however it has numbers of problems. Some major advantages is using too much memory from system,
+- **[BlueJ](http://www.bluej.org/)** is a simple editor mostly used to teach java and object oriented programming. The main advantage of the editor over other ones is that it does not need a main method to run the program.
+
+## Online IDEs
+
+- **[CodeRun Studio](http://www.webappers.com/2009/12/04/coderun-studio-free-cross-platform-online-ide/)** is an online IDE that you build web application. It allows you to run and test code online.
+
```

Comment:
We could, I didn't change much from the original as I wanted him to author it. I personally never heard of nitrous.io